GoVizzy/GoVizzy.ipynb

```diff
@@ -44,7 +44,7 @@
        "version_minor": 0
       },
       "text/plain": [
-       "AppLayout(children=(HBox(children=(Button(button_style='danger', description='[X]', layout=Layout(margin='0 0 …"
+       "AppLayout(children=(HBox(children=(Button(button_style='danger', description='[X]', layout=Layout(margin='0 0 �"
```

Comment:
On these lines you've put in some type of non-UTF8 character (in this line after the second 0). I tried running this locally and Jupyter refused to even open the file because it wasn't UTF8 encoded.